doc/admin-guide/files/ssl_server_name.config.en.rst

```diff
@@ -0,0 +1,124 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../../common.defs
+
+ssl_server_name.config
+**********************
+
+.. configfile:: ssl_server_name.config
+
+Description
+===========
+
+This file is used to configure aspects of TLS connection handling for both inbound and outbound
+connections. The configuration is driven the SNI values provided by the inbound connection.
```

Comment:
driven *by* the ?

Can we give some examples of what this does? 
For instance, this file can disable HTTP/2.0 for a particular domain or disable origin server certificate validation.

Reply:
I fixed the grammar, but precisely those examples are in the `Examples` section.